crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -29,8 +29,8 @@ fn prepare_world_normal(
     is_front: bool,
 ) -> vec3<f32> {
     var output: vec3<f32> = world_normal;
-#ifndef VERTEX_TANGENTS
-#ifndef STANDARDMATERIAL_NORMAL_MAP
+    #ifndef VERTEX_TANGENTS
+    #ifndef STANDARDMATERIAL_NORMAL_MAP
```

Comment:
Minor but I think all preprocessor stuff should be left-justified. Maybe it's just a habit from C preprocessor days but it makes it clearer in my opinion. I assume someone else has commented about indenting this?

Reply:
```suggestion
#ifndef VERTEX_TANGENTS
#ifndef STANDARDMATERIAL_NORMAL_MAP
```